### PR TITLE
[FISH-1034] - Payara 5.2021.1 Release Notes

### PR DESCRIPTION
**Pull request by cubastanley**
Release Notes for Payara 5.2021.1 Community Edition Release

TCK results will come later as a separate PR

### CODE REVIEW

**Review by AlanRoth — APPROVED** [review]

File: docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* [https://github.com/payara/Payara/pull/5020[FISH-994]] - [Community - https://github.com/sgflt[sgflt]] Package jaxws opentracing to embedded Payara  
+* [https://github.com/payara/Payara/pull/5082[FISH-1007]] - [Community - https://github.com/sgflt[sgflt]] Payara logback libs produce NPE 
+* [https://github.com/payara/Payara/pull/5071[FISH-1015]] - [Community - https://github.com/avpinchuk[avpinchuk]] Add appropriate application name to GAV deployments 
+* [https://github.com/payara/Payara/pull/4441[FISH-1017]] - [Community - https://github.com/bjetal[bjetal]] Payara can close JarFile instances used by current URLClassLoaders 
+* [https://github.com/payara/Payara/pull/5102[FISH-1018]] - Class Loader leaks on redeploy 
+* [https://github.com/payara/Payara/pull/5101[FISH-1019]] - [Community - https://github.com/bhanuurivalled[bhanuunrivalled]] GlassFishProperties NPE when initialised with null properties
+
+== Component Upgrades
+
+* [https://github.com/payara/Payara/pull/5014[FISH-791]] - Upgrade to Hazelcast 4
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc:70]: We should add a paragraph at the top that due to this upgrade of Hazelcast, rolling updates are not supported.

File: docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
+= Payara Community 5.2021.1 Release Notes
+
+== Supported APIs and Applications
+
+* Jakarta EE 8
+* Java EE 8 Applications
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc:6]: Add P 4.0 also here as supported API.

**Review by rdebusscher — CHANGES_REQUESTED** [review]

**Review by rdebusscher — APPROVED** [review]

File: docs/modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -295,6 +295,7 @@
 
 .Appendices
 * xref:release-notes/release-notes-history.adoc[History of Release Notes]
+** xref:release-notes/release-notes-2021-1.adoc[Payara Community 5.2021.1 Release Notes]
```
Comment (OndroMih) [inline docs/modules/ROOT/nav.adoc:298]: No need to duplicate the current release notes page here. Antora doesn't treat duplicate links nicely and would always jump in the sidebar to the page in the Overview even if users click on the link in the History of Release Notes section
```suggestion
```

File: docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* MicroProfile 4.0
+
+== Notes
+
+=== MicroProfile 4.0
+
+Following the 5.2020.7 release, we’ve upgraded our MicroProfile implementations to be compatible with the specifications present in version 4.0 of MicroProfile. You should be able to test your MicroProfile 4.0 compatible applications using this Community release. If you discover any issues, or have any related questions, please raise them on our GitHub repository.
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc:13]:
```suggestion
Following the 5.2020.7 release, we’ve upgraded Payara Platform to be compatible with the specifications present in version 4.0 of MicroProfile. You should be able to test your MicroProfile 4.0 compatible applications using this Community release. If you discover any issues, or have any related questions, please raise them on our GitHub repository. 
```

**Review by OndroMih — CHANGES_REQUESTED** [review]

**Pull request merged**